emulator/script_test.go

```diff
@@ -157,19 +157,82 @@ func TestInfiniteScript(t *testing.T) {
 
 	t.Parallel()
 
-	const limit = 1000
+	const limit = 90
 	b, err := emulator.New(
 		emulator.WithScriptGasLimit(limit),
 	)
 	require.NoError(t, err)
 
 	const code = `
-      pub fun main() {
-          main()
-      }
-    `
+	    pub fun main() {
+	        main()
+	    }
+	`
 	result, err := b.ExecuteScript([]byte(code), nil)
 	require.NoError(t, err)
 
 	require.True(t, fvmerrors.IsComputationLimitExceededError(result.Error))
 }
+
+func TestScriptWithExceeedingComputationLimit(t *testing.T) {
+
+	t.Parallel()
+
+	const limit = 2000
+	b, err := emulator.New(
+		emulator.WithScriptGasLimit(limit),
+	)
+	require.NoError(t, err)
+
+	const code = `
+	    pub fun main() {
```

Comment:
maybe store the script in a `const`

Reply:
Good catch, I restructured the tests to remove the script duplication in https://github.com/onflow/flow-emulator/pull/451/commits/82b2bd66f3812c0e9f82239a0ae218146157c0da